src/LearningObjects/LearningObjectInteractor.ts

```diff
@@ -938,15 +941,15 @@ export async function updateLearningObject({
  * @param {UserToken} requester [The requester of the releasable Learning Object]
  * @returns {Promise<LearningObject>}
  */
-async function generateReleasableLearningObject({
+export async function generateReleasableLearningObject({
```

Comment:
Do we need this function exported?
```suggestion
async function generateReleasableLearningObject({
```

Reply:
In order to test whether or not this object is returning the full hierarchy of a Learning Object I thought it'd be necessary to export this function to make it available in the spec file.